applications/serial_lte_modem/doc/TCPIP_AT_commands.rst

```diff
@@ -0,0 +1,1541 @@
+.. _SLM_AT_TCP_UDP:
+
+TCP and UDP AT Commands
+***********************
+
+The following commands list contains TCP and UDP related AT commands:
+
+* `BSD socket #XSOCKET`_
+* `BSD socket options #XSOCKETOPT`_
+* `Socket binding #XBIND`_
+* `TCP connection #XCONNECT`_
+* `TCP set listen mode #XLISTEN`_
+* `TCP accept incoming connection #XACCEPT`_
+* `Send data #XDATA`_
+* `Receive data #XRECV`_
+* `UDP send data #XSENDTO`_
+* `UDP receive data #XRECVFROM`_
+* `Resolve hostname #XGETADDRINFO`_
+* `TCP server #XTCPSVR`_
+* `TCP/TLS client #XTCPCLI`_
+* `TCP send data #XTCPSEND`_
+* `TCP receive data #XTCPRECV`_
+* `UDP server #XUDPSVR`_
+* `UDP/DTLS client #XUDPCLI`_
+* `UDP send data #XUDPSEND`_
+
+For more information on the BSD networking services, visit the `BSD Networking Services Spec Reference`_.
+
+BSD socket #XSOCKET
+===================
+
+The ``#XSOCKET`` command allows you to open or close a socket, or to check the socket handle.
+
+Set command
+-----------
+
+The set command allows you to open or close a socket.
+
+Syntax
+~~~~~~
+
+::
+
+   #XSOCKET=<op>[,<type>,<role>[,<sec_tag>]]
+
+* The ``<op>`` parameter can accept one the following values:
+
+  * ``0`` - Open
+  * ``1`` - Close
+
+* The ``<sec_tag>`` parameter is used in this AT command to indicate to the modem which credentials to use.
+  It is associated with the certificate or PSK.
+  Specifying the ``<sec_tag>`` is mandatory when opening a socket.
+  When TLS/DTLS is expected, the credentials should be stored on the modem side by ``AT%XCMNG`` or by the Nordic nRF Connect/LTE Link Monitor tool.
+  The modem needs to be in the offline state.
```

Comment:
Please verify if lines 54, 55, and 56 are actually correct.

Reply:
Yes, correct

Reply:
@junqingzoe the XCMNG  is not mentioned anywhere else, but being an "X" command I suppose it is one of the proprietary ones. Could you provide enough details on the command to actually add it in the list?

Reply:
@FrancescoSer This is one of the "normal" nRF91 AT commands: https://infocenter.nordicsemi.com/topic/ref_at_commands/REF/at_commands/mob_termination_ctrl_status/cmng.html

Reply:
Yes "X" means Nordic proprietary on modem side